lib/ansiblelint/rules/__init__.py

```diff
@@ -221,8 +221,7 @@ def run(self, playbookfile, tags=set(), skip_list=frozenset()) -> List:
                     matches.extend(rule.matchlines(playbookfile, text))
                     matches.extend(rule.matchtasks(playbookfile, text))
                     matches.extend(rule.matchyaml(playbookfile, text))
-
-        return matches
+        return list(set(matches))
```

Comment:
This is what I'm concerned about here https://github.com/ansible/ansible-lint/pull/881#discussion_r451631163

Reply:
Sorted